modules/reactor/src/vectorpostprocessors/IntegralReportingIDVPP.C

```diff
@@ -0,0 +1,165 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "IntegralReportingIDVPP.h"
+
+#include "MooseVariable.h"
+#include "MooseMesh.h"
+
+#include "libmesh/mesh_base.h"
+
+registerMooseObject("ReactorApp", IntegralReportingIDVPP);
+
+InputParameters
+IntegralReportingIDVPP::validParams()
```

Comment:
Can this object not be more general? I think something collecing values based on element integer may be valid in the farmework

Reply:
Also curious if you guys have ever looked at the Reporter system. It enables outputting arbitrary JSON. May prove useful once more output is desired.

Reply:
I am not aware of the `reporter system`. I will take a look. Thanks.